backend/business-logic/users.js

```diff
@@ -73,20 +72,16 @@ const userManager = {
       throw new Error(`Could not find such a user!`);
     }
   },
-  getAllUsers: async () => {
-    const usersAll = await userStore.all();
-    console.log(`GETALLuserS: ${usersAll}`);
-    if (!usersAll) {
-      throw new Error(`Could not find any users!`);
+  getAllVolunteers: async () => {
+    try {
+      const volunteers = await User.find(userType === "volunteer");
+      if (userData.userType === "") {
```

Comment:
here I guess you wanted to actually check if there are no volunteers... which would be volunteers ===